src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import Button from './Button';
+import Display from './Display';
+
+
+class App extends Component {
+  state = {
+    count: 0
+  }
+
+  incrementCount = () => {
+    const { count } = this.state;
+    this.setState({ count: count + 1 });
+  }
+
+  decreaseCount = () => {
+    const { count } = this.state;
+    if (this.state.count !== 0) {
```

Comment:
Hölgyem, ha a counter-t véletlen mínusz számra írnánk, akkor is működne, kérem javítani!